subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultTransformationRegistrationFactory.java

```diff
@@ -201,10 +208,19 @@ public TransformationStep getTransformationStep() {
 
     private static class NormalizerCollectingVisitor extends PropertyVisitor.Adapter {
         private Class<? extends FileNormalizer> normalizer;
+        private DirectorySensitivity directorySensitivity = DirectorySensitivity.DEFAULT;
 
         @Override
-        public void visitInputFileProperty(String propertyName, boolean optional, boolean skipWhenEmpty, boolean incremental, @Nullable Class<? extends FileNormalizer> fileNormalizer, PropertyValue value, InputFilePropertyType filePropertyType) {
+        public void visitInputFileProperty(String propertyName,
+                                           boolean optional,
+                                           boolean skipWhenEmpty,
+                                           DirectorySensitivity directorySensitivity,
+                                           boolean incremental,
+                                           @Nullable Class<? extends FileNormalizer> fileNormalizer,
+                                           PropertyValue value,
+                                           InputFilePropertyType filePropertyType) {
```

Comment:
```suggestion
        public void visitInputFileProperty(
            String propertyName,
            boolean optional,
            boolean skipWhenEmpty,
            DirectorySensitivity directorySensitivity,
            boolean incremental,
            @Nullable Class<? extends FileNormalizer> fileNormalizer,
            PropertyValue value,
            InputFilePropertyType filePropertyType
        ) {
```

Nit: I think it looks much nicer when the first argument and the closing bracket is on a new line.

There are a few more method calls/constructors where the first argument is on the same line as the opening bracket, I didn't comment on any of those.